ajax/libs/ngprogress/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "author": {
+    "name": "Victor Bjelkholm"
+  },
```

Comment:
Let's use `"author": "Victor Bjelkholm"` here. You can take a look at its package.json in npm package.